pibooth/view/background.py

```diff
@@ -9,6 +9,7 @@
 ARROW_TOP = 'top'
 ARROW_BOTTOM = 'bottom'
 ARROW_HIDDEN = 'hidden'
+TOUCHSCREEN = 'touchscreen'
```

Comment:
ARROW_TOUCHSCREEN or ARROW_TOUCH

(keep same prefix in variable to keep in mind the effet and where it is used)